### PR TITLE
Remove predeclared imports for decimal, string, float, int, error and xml

### PR DESCRIPTION
**Pull request by dulajdilshan**
## Purpose
$title

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/crypto/crypto.bal [hunk 1]
```diff
@@ -140,19 +139,20 @@ function encrypt() returns error? {
     [crypto:PrivateKey, crypto:PublicKey] [privateKey, publicKey] =
         check decodeKeys();
 
-    // Encrypt and decrypt an input value using RSA ECB PKCS1 padding.
+    // Encrypt and decry
+    pt an input value using RSA ECB PKCS1 padding.
```
Comment (MaryamZi) [inline examples/crypto/crypto.bal:143]: Unnecessary change?

```suggestion
    // Encrypt and decrypt an input value using RSA ECB PKCS1 padding.
```

Comment (dulajdilshan) [reply examples/crypto/crypto.bal:143]: Oh my bad

**Review by MaryamZi — APPROVED** [review]
